Sources/core/objects/Tile.cpp

```diff
@@ -418,3 +418,41 @@ void CubeTile::LoadInMap()
 {
     GetGame().GetMap().GetSquares()[posx_][posy_][posz_] = GetId();
 }
+
+
+
+bool CubeTile::Targetable(id_ptr_on<IOnMapBase> item) const
+{
+    if (!item.valid())
```

Comment:
This function does not use any of `CubeTile` variable and also it does not use any members of class. It should be made static and it should be made part of `Gun` class (or some suitable class for that case).